### DISCUSSION TITLE
Was the "Checkout Pull Request" removed?

### DISCUSSION THREAD

**Discussion by Linuus** [NeogitOrg/neogit#1123] [Q&A]
Wasn't there a "Checkout pull request" option before? Like `b n` or something? :)

![Screenshot 2024-01-11 at 09 47 47](https://github.com/NeogitOrg/neogit/assets/379823/8ef8b768-395b-47d0-adf9-80b540924bb8)

Or did I dream this? If not, how do I do it now?

**Comment by CKolkey** [comment]
You've gotta lay off the mushrooms, my dude :)

Alright, in all seriousness, no, there has never been a "checkout pull request", but also... that doesn't really make sense. A pull request is just a branch - or, more specifically, it's a **request** to a remote repository to **pull** changes from your branch, **merging** them into the remote branch. 

If what you're trying to do is checkout a branch from a remote, then you can do that via `local branch` - just select a remote branch like `origin/my-cool-feature`, and it'll create a local branch tracking that as it's upstream.

**Reply by Linuus** [reply]
Hah, no mushrooms here, I'm probably just misremembering. This feature is in Magit (or [forge](https://github.com/magit/forge) more specifically) which I use in Emacs so probably why I'm confused.

Checkout > pull-request

![Screenshot 2024-01-11 at 12 33 06](https://github.com/NeogitOrg/neogit/assets/379823/6f3031b5-3f07-44fc-88dd-e3f2493f9c54)

**Reply by CKolkey** [reply]
Ah, ok! I haven't checked out the stuff forge adds - still working on stock magit features. It looks cool, I'm definitely open to it.
